alignak/acknowledge.py

```diff
@@ -68,6 +69,7 @@ class Acknowledge(AlignakObject):  # pylint: disable=R0903
         'end_time': None,
         'author': None,
         'comment': None,
+        'comment_id': StringProp(default='')
```

Comment:
But a comment uuid is not a StringProp but a Python string ... as other alignak objects uuid